### PR TITLE
added 5 new questions to data.js

### PR DESCRIPTION
**Pull request by majidireshad**
# Pull Request

Added 5 new questions to data.js

## Checklist

Complete these things in order to submit your PR for review:

- [x] :mag: I have manually checked the code in my changes
- [ ] :judge: I have tested my changes
- [ ] :pencil: I have provided comments in code to explain my changes

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -116,6 +116,108 @@ export const quizData = {
           href: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/About_JavaScript',
         },
       ],
+    },
+    {
+      text: 'Where is the correct place to insert a JavaScript?',
```
Comment (cgduncan7) [inline src/data.js:121]:
```suggestion
      text: 'Where is the correct place to insert JavaScript in HTML?',
```

File: src/data.js [hunk 2]
```diff
@@ -116,6 +116,108 @@ export const quizData = {
...
+          href: 'https://www.javatpoint.com/how-to-add-javascript-to-html#:~:text=To%20add%20the%20JavaScript%20code%20into%20the%20HTML,of%20the%20web%20page%20that%20the%20users%20use.',
+        },
+        {
+          text: 'W3school',
+          href: 'https://www.w3schools.com/js/js_whereto.asp',
+        },
+      ],
+    },
+    {
+      text: ' --------------- method returns the number of milliseconds in a date string.',
```
Comment (cgduncan7) [inline src/data.js:141]:
```suggestion
      text: 'Which method returns the number of milliseconds in a date string?',
```

File: src/data.js [hunk 3]
```diff
@@ -116,6 +116,108 @@ export const quizData = {
...
+          href: 'https://www.javatpoint.com/java-date-parse-method#:~:text=Java%20Date%20parse%20%28%29%20Method%20The%20parse%20%28%29,1970%2C%2000%3A00%3A00%20GMT%20represented%20by%20this%20string%20argument.',
+        },
+        {
+          text: 'MDN',
+          href: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse',
+        },
+      ],
+    },
+    {
+      text: 'Who invented the javascript programming language?',
```
Comment (cgduncan7) [inline src/data.js:161]:
```suggestion
      text: 'Who invented the JavaScript programming language?',
```

File: src/data.js [hunk 4]
```diff
@@ -116,6 +116,108 @@ export const quizData = {
...
+          href: 'https://askingthelot.com/who-introduced-javascript/',
+        },
+        {
+          text: 'W3school JavaScript History',
+          href: 'https://www.w3schools.in/javascript-tutorial/history-of-javascript/',
+        },
+      ],
+    },
+    {
+      text: 'Math.round(-20.5)=?',
```
Comment (cgduncan7) [inline src/data.js:181]:
```suggestion
      text: 'Math.round(-20.5) returns what?',
```

**Review by cgduncan7 — CHANGES_REQUESTED** [review]

File: src/data.js [hunk 5]
```diff
@@ -116,6 +116,108 @@ export const quizData = {
...
+          href: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round',
+        },
+        {
+          text: 'W3school',
+          href: 'https://www.w3schools.com/jsref/jsref_round.asp',
+        },
+      ],
+    },
+    {
+      text: 'How do you delete an element from an options array?',
```
[the remark this answers is not in the document]
Comment (majidireshad) [reply src/data.js:202]: I might have not used the best source for questions. I replace this one with another one.

File: src/data.js [hunk 6]
```diff
@@ -116,6 +116,108 @@ export const quizData = {
...
+    },
+    {
+      text: 'Which of the following looping structures are available in javaScript?',
+      answers: {
+        a: 'for, forEach',
+        b: 'forEach, while loop',
+        c: 'do...while loop,forEach',
+        d: 'for , while loop',
+      },
+      correct: 'd',
```
Comment (cgduncan7) [inline src/data.js:209]:
```javascript
let x = 0
do {
  console.log(x++)
} while (x < 10)
```

is available in JavaScript 🙃

Comment (cgduncan7) [reply src/data.js:209]: Maybe you could ask the team if they can help provide some possible questions?

**Review by cgduncan7 — CHANGES_REQUESTED** [review]

**Comment by cgduncan7** [comment]
Closing in favor of https://github.com/cgduncan7/browser-quiz-project-starter/pull/13

**Pull request closed without merging**